packages/synapse-react-client/src/synapse-client/SynapseClientUtils.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  ErrorResponseCode,
+  TwoFactorAuthErrorResponse,
+} from '@sage-bionetworks/synapse-types'
+import { SynapseClientError } from '../utils/SynapseClientError'
+
+export function isOutsideSynapseOrg() {
+  return !window.location.hostname.toLowerCase().endsWith('.synapse.org')
+}
```

Comment:
This was a constant, but now it's a function (so I can more easily mock it in the SynapseClient tests!)